etc/ReplayOfflineConfiguration.py

```diff
@@ -1563,6 +561,37 @@
 ignoreStream(tier0Config, "streamL1Rates")
 ignoreStream(tier0Config, "streamDQMRates")
 
+############################################
+### ignored streams for the current test ###
+############################################
+ignoreStream(tier0Config, "ALCALUMIPIXELS")
+ignoreStream(tier0Config, "ALCALUMIPIXELSEXPRESS")
+ignoreStream(tier0Config, "ALCAP0")
+ignoreStream(tier0Config, "ALCAPHISYM")
+ignoreStream(tier0Config, "Calibration")
+ignoreStream(tier0Config, "Express")
+ignoreStream(tier0Config, "ExpressAlignment")
+ignoreStream(tier0Config, "ExpressCosmics")
+ignoreStream(tier0Config, "ExpressPA")
+ignoreStream(tier0Config, "HIExpress")
+ignoreStream(tier0Config, "HIExpressAlignment"
+ignoreStream(tier0Config, "HLTMonitor")
+ignoreStream(tier0Config, "HLTMonitorPA")
+ignoreStream(tier0Config, "NanoDST")
+ignoreStream(tier0Config, "ParkingBPH1")
+ignoreStream(tier0Config, "ParkingBPH2")
+ignoreStream(tier0Config, "ParkingBPH3")
+ignoreStream(tier0Config, "ParkingBPH4")
+ignoreStream(tier0Config, "ParkingBPH5")
+ignoreStream(tier0Config, "PhysicsCommissioning")
```

Comment:
The streamer `PhysicsCommissioning` needs to be included in order to generate PDs such us `ZeroBias`

Reply:
ah ok, makes sense! i'll remove this line